PageViews/tests/PageViews-specs.js

```diff
@@ -0,0 +1,164 @@
+import '@testing-library/jest-dom';
+import {render, screen, waitFor} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import {PageViews, Page} from '../';
+
+describe('PageViews Specs', () => {
+	test(
+		'should show next button on the first page',
+		() => {
+			render(
+				<PageViews index={0}>
+					<Page>I gots contents</Page>
+					<Page>I gots contents</Page>
+				</PageViews>
+			);
+
+			const nextButton = screen.getByLabelText('Next');
```

Comment:
Let's test if the previous button is hidden on the first page.

Reply:
It's added.